community/php7/APKBUILD

```diff
@@ -535,6 +539,14 @@ phar() {
 	fi
 }
 
+default_exts() {
+	pkgdesc="PHP$_suffix metapackage for all Core Extensions and Bundled Extensions"
+	depends=$(printf '%s\n' $_extensions \
+		| sed -En "s/(\w+)\*/$pkgname-\1/p")
+
+	mkdir -p "$subpkgdir"
+}
```

Comment:
I’m afraid that _php7-bundled_ is very misleading name. _php7-bundled-exts_ would be a bit better, but still misleading, because they are in fact not bundled. I don’t like _php7-default-exts_ either, but can’t figure out anything better.